packages/@office-iss/react-native-win32/src/Libraries/Utilities/FocusManager.win32.js

```diff
@@ -21,17 +21,15 @@ class FocusManager {
   static focus(ref: React.Ref<any>, setWindowFocus: boolean) {
     if (ref) {
       if (setWindowFocus) {
-        NativeModules.UIManager.dispatchViewManagerCommand(
+        UIManager.dispatchViewManagerCommand(
           findNodeHandle(ref),
-          NativeModules.UIManager.getViewManagerConfig('RCTView').Commands
-            .aggressivefocus,
+          UIManager.getViewManagerConfig('RCTView').Commands.aggressivefocus,
           null,
         );
       } else {
-        NativeModules.UIManager.dispatchViewManagerCommand(
+        UIManager.dispatchViewManagerCommand(
           findNodeHandle(ref),
-          NativeModules.UIManager.getViewManagerConfig('RCTView').Commands
-            .politefocus,
+          UIManager.getViewManagerConfig('RCTView').Commands.politefocus,
```

Comment:
A bit OT, but what all happens under the hood when we're grabbing a VM constant? Are they cached JS-thread side or is it a stop the world situation?

Reply:
https://github.com/microsoft/react-native-windows/blob/4b744d15dc915a805405db8184cfb863c2a24e61/packages/%40office-iss/react-native-win32/src/Libraries/ReactNative/PaperUIManager.win32.js#L34
Its cached on the JS side.